packages/cron/src/jobs/storage.js

```diff
@@ -1,74 +1,99 @@
 import debug from 'debug'
-import { EmailService } from '../lib/email.js'
 import { EMAIL_TYPE } from '@web3-storage/db'
 
 const log = debug('storage:checkStorageUsed')
-let emailService
+
+const STORAGE_QUOTA_EMAILS = [
+  {
+    emailType: EMAIL_TYPE.User100PercentStorage,
+    fromPercent: 100,
+    secondsSinceLastSent: 60 * 60 * 23 // 1 day approx
+  },
+  {
+    emailType: EMAIL_TYPE.User90PercentStorage,
+    fromPercent: 90,
+    toPercent: 100,
+    secondsSinceLastSent: 60 * 60 * 23 // 1 day approx
+  },
+  {
+    emailType: EMAIL_TYPE.User85PercentStorage,
+    fromPercent: 85,
+    toPercent: 90,
+    secondsSinceLastSent: 60 * 60 * 24 * 7 // 1 week
```

Comment:
```suggestion
    secondsSinceLastSent: 60 * 60 * 24 * 7 - 60 * 60 // 1 week, with flexibility for cron irregularity
```

I think that all of these need the ~1 hour flexibility on them. Otherwise, if the cron doesn't run exactly daily then these emails will sometimes get sent every 8 days rather than every 7.

Reply:
Have added the flex throughout.